### PR TITLE
:bug: (darkmode) fix more issues - transaction table and csv imports

### PR DESCRIPTION
**Pull request by MatissJanis**
Fixing even more darkmode regressions.

- `select` boxes in csv import modals were ugly (demo import file: [headerless.csv](https://github.com/actualbudget/actual/files/12383953/headerless.csv))
- the select checkbox in transaction table was not visible on hover
- cleaning up some unnecessary usage of `borderColor`

### CODE REVIEW

File: packages/desktop-client/src/components/budget/misc.js [hunk 1]
```diff
@@ -437,7 +437,6 @@ function SidebarCategory({
         formatter={value => displayed}
         width="flex"
         exposed={editing || temporary}
-        borderColor={dragPreview ? 'transparent' : borderColor}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/budget/misc.js:440]: I removed `borderColor` prop a few PRs ago. Just cleaning it up here.

File: packages/desktop-client/src/components/common/Select.tsx [hunk 2]
```diff
@@ -49,13 +49,6 @@ export default function Select<Value extends string>({
   disabledKeys = [],
 }: SelectProps<Value>) {
   const arrowSize = 7;
-  const checkHeight = !style
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/common/Select.tsx:52]: Changes in this file are style reverts to the `Select` component introduced here https://github.com/actualbudget/actual/commit/9fed15f88b0e124cfbd663a4b233ee32c12ec02f#diff-5f4290e10caa20f2b8c41fae4d8f3b0aa45662d695adab848fd19381279f5e5a

I did not revert the colors, so those should still be fine in darkmode.

File: packages/desktop-client/src/components/common/View.tsx [hunk 3]
```diff
@@ -22,7 +22,7 @@ const View = (props: ViewProps) => {
       {...restProps}
       ref={innerRef}
       style={nativeStyle}
-      className={`view ${css([props.className, props.style])}`}
+      className={`view ${props.className} ${css([props.style])}`}
```
Comment (MatissJanis) [inline packages/desktop-client/src/components/common/View.tsx:25]: `className` was not working before :)

Comment (joel-jeremy) [inline packages/desktop-client/src/components/common/View.tsx:25]:
```suggestion
      className={`view ${props.className} ${css(props.style)}`}
```
I think the array is no longer needed here? Other than that, looks good! :)

Comment (joel-jeremy) [reply packages/desktop-client/src/components/common/View.tsx:25]: Maybe this was the root cause of the issue I was having when I experimented with emotionjs.

**Comment by MatissJanis** [comment]
This PR is now ready for a full review.

**Review by joel-jeremy — APPROVED** [review]

**Pull request merged**